Core/Object Arts/Dolphin/Base/Deprecated/Dolphin Base (Deprecated).pax

```diff
@@ -1102,7 +1102,7 @@ unicodeClass
 
 asTimeStamp
 	Notification deprecated.
-	^self asDateAndTime asTime! !
+	^self asDateAndTime asTimeStamp.! !
```

Comment:
I'm not sure this is worth fixing, as it is deprecated. I'm even less sure it is worth fixing without a test. Still...